I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00keymap.rpy", line 473, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 473, in script
    python hide:
  File "renpy/common/00keymap.rpy", line 473, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 492, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
AttributeError: 'NoneType' object has no attribute 'stores'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\bootstrap.py", line 274, in bootstrap
    renpy.main.main()
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\main.py", line 548, in main
    if renpy.parser.report_parse_errors():
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\parser.py", line 3109, in report_parse_errors
    renpy.display.error.report_parse_errors(full_text, error_fn)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\display\error.py", line 195, in report_parse_errors
    renpy.game.invoke_in_new_context(
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\game.py", line 286, in invoke_in_new_context
    return callable(*args, **kwargs)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\display\error.py", line 48, in call_exception_screen
    return renpy.ui.interact(mouse="screen", type="screen", suppress_overlay=True, suppress_underlay=True)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\ui.py", line 299, in interact
    rv = renpy.game.interface.interact(roll_forward=roll_forward, **kwargs)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\display\core.py", line 3377, in interact
    repeat, rv = self.interact_core(preloads=preloads, trans_pause=trans_pause, pause=pause, pause_start=pause_start, **kwargs) # type: ignore
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\display\core.py", line 3867, in interact_core
    renpy.loader.check_autoreload()
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\loader.py", line 1157, in check_autoreload
    renpy.exports.reload_script()
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\exports.py", line 1843, in reload_script
    renpy.game.call_in_new_context("_save_reload_game")
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\game.py", line 346, in call_in_new_context
    return renpy.execution.run_context(False)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\execution.py", line 938, in run_context
    context.run()
  File "renpy/common/00keymap.rpy", line 473, in script
    python hide:
  File "/home/tom/ab/renpy-build/tmp/install.linux-x86_64/lib/python3.9/site-packages/future/utils/__init__.py", line 441, in raise_
  File "renpy/common/00keymap.rpy", line 473, in script
    python hide:
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\ast.py", line 1131, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\python.py", line 1049, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "renpy/common/00keymap.rpy", line 473, in <module>
    python hide:
  File "renpy/common/00keymap.rpy", line 492, in _execute_python_hide
    renpy.save("_reload-1", "reload save game")
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\loadsave.py", line 382, in save
    roots = renpy.game.log.freeze(None)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\rollback.py", line 1017, in freeze
    self.complete(False)
  File "C:\Users\student\Desktop\Applications\renpy-8.0.1-sdk\renpy\rollback.py", line 584, in complete
    self.current.stores[name], self.current.delta_ebc[name] = delta
AttributeError: 'NoneType' object has no attribute 'stores'

Windows-10-10.0.19043 AMD64
Ren'Py 8.0.1.22070801
AAA 0.0.3.20
Mon Dec 12 12:43:05 2022
